R/colourInput.R

```diff
@@ -72,11 +76,10 @@
 colourInput <- function(inputId, label, value = "white",
                         showColour = c("both", "text", "background"),
                         palette = c("square", "limited"),
-                        allowedCols,
+                        allowedCols, allowAlpha = FALSE,
```

Comment:
Adding the parameter in the middle would make this a breaking change. Anyone who currently uses colourInput without specifying the parameter names would assume the 7th parameter is `allowTransparent` , so this would cause problems for them.  It's not a good idea anyway to use 7 unnamed parameters, so you could argue it's their fault :) But I prefer adding new parameters at the end